datalad/distribution/tests/test_create_github.py

```diff
@@ -0,0 +1,61 @@
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Test create publication target github"""
+
+from os.path import join as opj
+# this must with with and without pygithub
+from datalad.api import create_sibling_github
+from datalad.api import Dataset
+from datalad.support.exceptions import MissingExternalDependency
+from datalad.tests.utils import with_tempfile
+from nose.tools import assert_raises, assert_in, assert_true, assert_false, \
+    assert_not_in, assert_equal
+from nose import SkipTest
+
+
+try:
+    import github as gh
+except ImportError:
+    # make sure that the command complains too
+    assert_raises(MissingExternalDependency, create_sibling_github, 'some')
```

Comment:
Lovely! Although I would not exercise assert right at import time -  make it into a new test_ function right here and nose will pick it up I i guess.